选修课程/计算机视觉/HITCV_EXP/exp3/exp-03-code/Unet/test_segmentation.ipynb

```
@@ -0,0 +1,366 @@
+    "          # Apply the same seed for consistency in augmentations\n",
+    "            seed = np.random.randint(0, 10000)\n",
+    "\n",
+    "            # Apply augmentations to the image and label\n",
+    "            torch.random.manual_seed(seed)\n",
+    "            image = self.same_augmentation(image)\n",
+    "            label = self.same_augmentation(label)\n",
+    "            # Reset the seed\n",
+    "            torch.random.manual_seed(seed)\n",
+    "\n",
+    "        # Apply transformations to the input image\n",
+    "        image = self.transform(image).to(self.device)\n",
```

Ruff (0.15.21)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)